pysph/sph/wc/linalg.py

```diff
@@ -62,7 +62,8 @@ def mat_vec_mult(a=[1.0, 0.0], b=[1.0, 0.0], n=3, result=[0.0, 0.0]):
         result[i] = s
 
 
-def augmented_matrix(A=[0.0, 0.0], b=[0.0, 0.0], n=3, na=1, result=[0.0, 0.0]):
+def augmented_matrix(A=[0.0, 0.0], b=[0.0, 0.0], n=3, na=1, nmax=3,
```

Comment:
What is nmax, it should be described in the docstring and clearly explained.  Why and where is this needed?

Reply:
done